dbt_utils_integration_tests_howto/integration_tests__models__web__test_url_host__sql.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/integration_tests/models/web/test_url_host.sql b/integration_tests/models/web/test_url_host.sql
+index 06a9f69..4ad5da5 100644
+--- a/integration_tests/models/web/test_url_host.sql
++++ b/integration_tests/models/web/test_url_host.sql
+@@ -1,3 +1,6 @@
++{{
```

Comment:
Please explain, why materialization is changed to type "table"?
We have noticed the same in multiple patch files.

Reply:
views don't like calculations where the output of a column is a manipulation of multiple long varchars, whereas tables are fine with that. This is a way to workaround this limitation. I think it does not change the semantic of the test